src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -243,7 +243,7 @@ public Tensor ProcessImage(string path)
                     var imageTensor = imageProcessor.ProcessImage(imagePathStr);
                     runner.AddInput(imageTensor, 0);
                     var featurizedImage = runner.Run()[0]; // Reuse memory?
-                    writer.WriteLine(label - 1 + "," + string.Join(",", featurizedImage.Data<float>()));
+                    writer.WriteLine(label - 1 + "," + string.Join(",", featurizedImage.ToArray<float>()));
```

Comment:
The same comment as above about using `ToArray`. It is usually an indication of a performance problem.

Reply:
Replied there.

---
In reply to: [323476010](https://github.com/dotnet/machinelearning/pull/4205#discussion_r323476010) [](ancestors = 323476010)